docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,328 @@
+# Prometheus Time Series Database migration
+This article provides a set of guidelines to migrate the time series database (TSDB from now on) from an existing Prometheus deployment to the Bitnami Kubernetes Production Runtime one.
+
+## Prerequisites
+The following prerequisites should be met before starting with the migration process:
+
+* You ave a healthy Kubernetes cluster running.
+* You have a Prometheus deployment that *matches the BKPR Prometheus version*, which can be obtained [here](https://github.com/bitnami/kube-prod-runtime#release-compatibility).
+
+This article will not cover the upgrade process of Prometheus itself. If you need to update your Prometheus deployment, check the [Prometheus docs](https://prometheus.io/docs/prometheus/latest/migration/)
+
+## Overview
+
+This chapter provides an overview of the required steps to migrate the Prometheus TSDB from an already running deployment to the BKPR Prometheus one.
+
+This process is meant to provide a zero downtime migration to the new StatefulSet by having two Prometheus instances running at the same time. These instances will be modified to include a sidecar container that will deploy an *rsync* client and server with access to the Prometheus data volume in order to move a Prometheus TSDB snapshot from one pod to another.
+
+As Prometheus doesn’t include yet a *flush* method to dump all the WAL file that is being written into a snapshot with a block format that doesn’t overlap time ranges (see [this issue](https://github.com/prometheus/tsdb/issues/346) for more context) it is necessary to sync with the Prometheus TSDB compact times, in order to ensure a minimal data loss (less than 5 minutes).
+
+These are the steps that will be made to accomplish this migration:
+
+* Deploy rsync sidecar container (acting as the rsync server) in your current Prometheus deployment 
+* Modify the `kubeprod-manifest.jsonnet` to add another sidecar rsync container (acting as a client)
```

Comment:
Modify the `kubeprod-manifest.jsonnet` file

Reply:
done.